docs/site_specific_config/rocm.md

```diff
@@ -0,0 +1,509 @@
+# Overview of ROCm Ecosystem (v6.4.1-20250526)
+
+> ⚠️ This document is a work-in-progress.
+> It may still contain inaccuracies or mistakes.
+>
+> This overview is being created in the context of adding support for ROCm to EESSI, the European Environment for Scientific Software Installations (https://eessi.io).
+>
+> Last update: 26 May 2025
```

Comment:
use an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/):
```suggestion
!!! warning "Work-in-progress"
    This document is a work-in-progress.
    It may still contain inaccuracies or mistakes.
    
    This overview is being created in the context of adding support for ROCm to EESSI, the European Environment for Scientific Software Installations (<https://eessi.io>).
    
    *Last update: 26 May 2025*